lib/js/src/manager/permission/_PermissionManagerBase.js

```diff
@@ -124,7 +125,7 @@ class PermissionManagerBase extends SubManagerBase {
      * @param {HMILevel} previousHmiLevel - The previous HMI Level enum value.
      * @param {PermissionItem[]} currentPermissionItems - An array of current PermissionItem instances.
      * @param {HMILevel} currentHmiLevel - The current HMI Level enum value.
-     * @returns {PermissionManagerBase} - A reference to this instance to support method chaining.
+     * @returns {_PermissionManagerBase} - A reference to this instance to support method chaining.
      */
     _notifyListeners (previousPermissionItems, previousHmiLevel, currentPermissionItems, currentHmiLevel) {
         for (const filter of this._filters) {
```

Comment:
getPermissionElements to _getPermissionElements within this function. same with getGroupType